src/components/orgunits/OrgUnitFieldSelect.js

```diff
@@ -34,9 +34,9 @@ const OrgUnitFieldSelect = () => {
     const attribute = attributes.find((a) => a.id === orgUnitField)
```

Comment:
Maybe this file should be renamed to AssociatedGeometrySelect

Reply:
Fixed.